logback-classic/src/main/java/ch/qos/logback/classic/ClassicConstants.java

```diff
@@ -28,6 +28,9 @@ public class ClassicConstants {
             + "comp/env/logback/configuration-resource";
     public static final String JNDI_CONTEXT_NAME = JNDI_JAVA_NAMESPACE + "comp/env/logback/context-name";
 
+    /** JNDI name of custom contextSelector classname, if used */
+    public static final String JNDI_LOGBACK_CONTEXT_SELECTOR = "java:comp/env/logback/contextSelector";
```

Comment:
This constant should match existing coding conventions, just like `JNDI_CONFIGURATION_RESOURCE` and `JNDI_CONTEXT_NAME`
```suggestion
    public static final String JNDI_LOGBACK_CONTEXT_SELECTOR = JNDI_JAVA_NAMESPACE + "comp/env/logback/contextSelector";
```

Reply:
Yes makes total sense

Reply:
Should I resubmit the request?